src/app/companies/page.tsx

```diff
@@ -1,12 +1,32 @@
 import HeaderLayout from "~/components/header-layout";
+import { RoleReviewCard } from "~/components/role-review-card";
 import SearchFilter from "~/components/search-filter";
+import { api } from "~/trpc/server";
+import { unstable_noStore as noStore } from "next/cache";
 
 export default async function Companies() {
+  /**
+   * FIXME: This is a temporary fix, figure out how to get build command working without noStore();
+   * @returns A promise containing the roles from the database
+   */
+  async function getRoles() {
+    noStore();
+    const roles = await api.role.list.query();
+    return roles;
+  }
+
+  const roles = await getRoles();
+
   return (
-    <div className="flex h-full flex-col">
-      <HeaderLayout>
-        <SearchFilter />
-      </HeaderLayout>
-    </div>
+    <HeaderLayout>
```

Comment:
Can we move the header layout to the `layout.tsx` file? We can have multiple `layout.tsx` files to have different layouts for groups of pages (since some pages don't use the same header)?

See: [Pages and Layouts](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts)